pkg/operators/limiter/limiter.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2024 The Inspektor Gadget authors
+// Copyright 2025 The Inspektor Gadget authors
```

Comment:
When updating it should be the year range
```suggestion
// Copyright 2024-2025 The Inspektor Gadget authors
```